packages/devextreme/js/__internal/ui/m_tag_box.ts

```diff
@@ -140,8 +140,8 @@ const TagBox = (SelectBox as any).inherit({
       rightArrow(e) {
         if (
           !this._isCaretAtTheStart()
-                    || this._isEmpty()
```

Comment:
Revert this formatting please. It doesn't matter for your PR

Reply:
[6a3a6dc](https://github.com/DevExpress/DevExtreme/pull/27063/commits/6a3a6dc948070b72983e42ce34c7c7d9f1c6b1aa)